### ISSUE TITLE
[Warrior][Talent] [Spell] Sudden Death Execute sometimes does no damage

### ISSUE THREAD

**Issue by Greywolf59** [WoWManiaUK/Redemption#6085]
Problem: If warrior gets [Sudden Death](https://www.wow-mania.com/armory/?spell=52437) proc there is pretty much regular chance the [Execute](https://www.wow-mania.com/armory/?spell=47471) will consume the charge and rage but won't do any damage. The occurence is circa 1 in 15 Executes. There are no floating damage numbers when used and Skada won't register the hit either. I am hitcapped, standing behind the enemy even though under Expertise cap I would notice if the Execute would be dodged (OP activation). I do believe that regular Executes under 20% HP are fine. I will have to test that (Where is 20%HP dummy for Horde?).
Update: Sometimes it does no damage but it does not consume the proc so the player ahs to use Execute twice to make it happen (GCD is activated).
INFO:
Toon: Lvl 80 orc warrior on Naxxramas realm
Talents+glyphs: LINK REMOVED
Set: 4pt9 (+6 % ARP in Battle Stance; +5 % crit for HS and Slam)

**Comment by fvicaria** [comment]
Please use our armory when describing issues and not another server's.

If this is "simetimes" it is a probability thing check the rank and the actual chance of the spell

**Comment by fvicaria** [comment]
Please use links to OUR armoury and not to another server's.

If this is "sometimes" it it most likely a probability/chance thing. Check spell rank and probability.

**Comment by fvicaria** [comment]
After checking this the probability is actually low. 3%, 6% and 9% depending on the rank.

**Comment by Greywolf59** [comment]
I am using your armory. I am not aware that you have your own talent calculator. Also the issue is that sometimes Execute does no damage. I know Sudden Death has 9% chance to proc from all attacks and it does. I also have found the time to test it a bit further and A: there is no problem with normal Execute under 20% HP. B: This problem does not occur in so called "Incite spec", only in the "Improved Execute" spec so the problem might be either with Glyph of Execute or Improved Execute talent.

**Comment by Echinaceax** [comment]
thanks @Greywolf59 for info we will take a look and see if we can reproduce it checking glyph etc..

**Comment by Greywolf59** [comment]
I think I have noticed this behaviour even on my low level alt without the glyph or Imp Exe talent (was levellingfor MPs). the fact that I did not reproduce this behaviour under % HP Execute might be linekd ot the fact I have tested it on that custom Wow-Mania dummy. I f I am not mistaken, this dummy can't dodge or even parry perhaps. Taht would mean the Execute does simply not register Dodges and Parries properly. Arms warriors do not use Expertise for dodge cap, they are under cap. However Execute should always proc Overpower from dodge. I had no dodge on Execute whatsoever, nor does Skada shows any dodge on Execute in any fights I have been through. Just throwing here another theory.
